packages/near-fast-auth-signer-e2e-tests/mpc-service.ts

```diff
@@ -0,0 +1,153 @@
+import { KeyPair } from '@near-js/crypto';
```

Comment:
I see some parts of it are copied from the app source code. Can we import it instead of copy? So we don't need to maintain duplicated code.

Reply:
Removed. 
It's actually useless now, it's already imported.